modules/nf-core/xengsort/index/tests/main.nf.test

```diff
@@ -0,0 +1,37 @@
+nextflow_process {
+
+    name "Test Process XENGSORT_INDEX"
+    script "../main.nf"
+    process "XENGSORT_INDEX"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "xengsort"
+    tag "xengsort/index"
+
+    test("homo_sapiens - sarscov2 - stub") {
```

Comment:
The inputs look feasible for running a normal minimal test dataset. Why isn't there one? I saw one had been added and then removed. Usually only running stub tests are for processes that require large database inputs.

Reply:
It doesn't work in github actions, you can rebase and run locally `nf-core modules test xengsort/index --profile docker` as I was doing and see it will works locally. But in github actions gives this error here: https://github.com/nf-core/modules/actions/runs/8434908174/job/23099202342

Also posted few comments on slack: https://app.slack.com/client/TE6CZUZPH/CE6SDBX2A

![image](https://github.com/nf-core/modules/assets/10541523/6e18c9d5-4c2d-4079-a6f0-b1db4d322a98)

I think it's a thread limitation from tool. I also tried to force 1 thread in all flags but it didn't work as well.

Reply:
I might come back to this test fix after implementing `xengsort/classify` but it looks like a limitation from github actions.